Applications/Spire/Source/Ui/OverlayPanel.cpp

```diff
@@ -97,21 +102,44 @@ void OverlayPanel::set_positioning(Positioning positioning) {
   m_positioning = positioning;
 }
 
+bool OverlayPanel::event(QEvent* event) {
+  if(m_is_closed_on_blur) {
+    if(event->type() == QEvent::WindowDeactivate) {
```

Comment:
What part of the spec does this implement?

Reply:
This was the main update in this case for the concept of a 'FocusOut'. If a child window of the OverlayPanel is activated then it's only considered a blur but not a FocusOut, so the OverlayPanel should stay open.

Reply:
Okay, use the recently merged ```FocusObserver``` to monitor the focus state of the ```OverlayPanel```. If you identify bugs in the ```FocusObserver```, report them.